File: SimpleGridview/app/src/main/java/com/mydemo/simplegridview/MainActivity.java
package com.mydemo.simplegridview;

import androidx.appcompat.app.AppCompatActivity;

import android.media.Image;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.GridView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        GridView gv =(GridView) findViewById(R.id.gridview);
        gv.setAdapter(new ImageAdapter( context: this));
        gv.setOnItemClickListener( new AdapterView.OnItemClickListener(){
            @Override
            public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
                Toast.makeText(context:MainActivity.this,text:"Image Position:"+position,Toast.makeText()
            }
        } );
    }
}

File: TextToSpeech/app/src/main/java/com/devendra/texttospeech/MainActivity.kt
package com.devendra.texttospeech

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.speech.tts.TextToSpeech
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var btnSpeak=findViewById<Button>(R.id.btn1)
        var editTextField=findViewById<EditText>(R.id.editTextText)

        tts= TextToSpeech(applicationContext.textto)

        btnSpeak.setOnClickListener{
            var.text=editTextField.text
            Toast.makeText(, this, Toast.LENGTH_SHORT).show()
        }
    }
}